Interesting error in Travis:
```
goroutine 1130 [running]:
github.com/status-im/status-go/geth/transactions/queue.(*TxQueue).Stop(0xc4208752c0)
	/home/travis/gopath/src/github.com/status-im/status-go/geth/transactions/queue/queue.go:89 +0x79
github.com/status-im/status-go/geth/transactions.(*Manager).Stop(0xc420775580)
	/home/travis/gopath/src/github.com/status-im/status-go/geth/transactions/txqueue_manager.go:72 +0x63
github.com/status-im/status-go/geth/api.(*StatusBackend).stopNode(0xc420875310, 0x0, 0x0)
	/home/travis/gopath/src/github.com/status-im/status-go/geth/api/backend.go:135 +0x60
github.com/status-im/status-go/geth/api.(*StatusBackend).StopNode(0xc420875310, 0x0, 0x0)
	/home/travis/gopath/src/github.com/status-im/status-go/geth/api/backend.go:128 +0x77
github.com/status-im/status-go/t/e2e/api_test.(*APIBackendTestSuite).TestRaceConditions.func2(0xc4222b9340)
	/home/travis/gopath/src/github.com/status-im/status-go/t/e2e/api/backend_test.go:56 +0x7c
created by github.com/status-im/status-go/t/e2e/api_test.(*APIBackendTestSuite).TestRaceConditions
	/home/travis/gopath/src/github.com/status-im/status-go/t/e2e/api/backend_test.go:175 +0x919
FAIL	github.com/status-im/status-go/t/e2e/api	45.402s
make: *** [test-e2e] Error 1
```

--- a/_assets/patches/geth/0016-fix-leveldb-issue.patch
+++ b/_assets/patches/geth/0016-fix-leveldb-issue.patch
@@ -1,3 +1,23 @@
+diff --git a/les/backend.go b/les/backend.go
+index 333df920..450d8351 100644
+--- a/les/backend.go
++++ b/les/backend.go
+@@ -20,7 +20,6 @@ package les
+ import (
+ 	"fmt"
+ 	"sync"
+-	"time"
+ 
+ 	"github.com/ethereum/go-ethereum/accounts"
+ 	"github.com/ethereum/go-ethereum/common"
+@@ -248,7 +247,6 @@ func (s *LightEthereum) Stop() error {
+ 
+ 	s.eventMux.Stop()
+ 
+-	time.Sleep(time.Millisecond * 200)
+ 	s.chainDb.Close()
+ 	close(s.shutdownChan)
+ 
 diff --git a/les/handler.go b/les/handler.go
 index 613fbb79..a6ae09ef 100644
 --- a/les/handler.go
@@ -21,7 +41,7 @@ index 613fbb79..a6ae09ef 100644
  	}
  	if odr != nil {
  		manager.retriever = odr.retriever
-@@ -210,9 +215,31 @@ func NewProtocolManager(chainConfig *params.ChainConfig, lightSync bool, protoco
+@@ -210,9 +215,32 @@ func NewProtocolManager(chainConfig *params.ChainConfig, lightSync bool, protoco
  		manager.fetcher = newLightFetcher(manager)
  	}
 
@@ -32,6 +52,7 @@ index 613fbb79..a6ae09ef 100644
 
 +func (pm *ProtocolManager) monitorDownloads() {
 +	sub := pm.eventMux.Subscribe(downloader.StartEvent{}, downloader.DoneEvent{}, downloader.FailedEvent{})
++	defer sub.Unsubscribe()
 +
 +	for {
 +		select {

